packages/url/native/URL.re

```diff
@@ -214,8 +214,12 @@ let search = url => {
   | _ => Some("?" ++ Uri.encoded_of_query(~scheme, query))
   };
 };
-let setSearch = (t, string) => {
-  Uri.with_query(t, Uri.query_of_encoded(string));
+let setSearchString = (t, searchString) => {
+  Uri.with_query(t, Uri.query_of_encoded(searchString));
+};
+let setSearch = (t, searchParams) => {
+  let queryString = SearchParams.toString(searchParams);
+  setSearchString(t, queryString);
```

Comment:
no need to transform toString here, Uri.with_query should do the job, right?

Reply:
Good point